backend/src/v5/processors/teamspaces/projects/models/containers.js

```diff
@@ -55,4 +56,18 @@ Containers.getContainerStats = async (teamspace, project, container) => {
 	};
 };
 
+Containers.getRevisions = async (teamspace, container, showVoid) => {
+	const revisions = await getRevisions(teamspace, container, showVoid,
+		{ _id: 1, author: 1, timestamp: 1, tag: 1, void: 1 });
+
+	return revisions.map((revision) => ({ ...revision, _id: UUIDToString(revision._id) }));
```

Comment:
do the conversion in routes.

this is serialisation, we only need it for network requests. serialising/deserialising should only happen in middleware (inputs) or routes(outputs)